utils/doclint/check_public_api/Documentation.js

```diff
@@ -59,14 +63,17 @@ Documentation.Member = class {
   /**
    * @param {string} kind
    * @param {string} name
-   * @param {!Documentation.Type} type
+   * @param {?Documentation.Type} type
    * @param {!Array<!Documentation.Member>} argsArray
    */
-  constructor(kind, name, type, argsArray) {
+  constructor(kind, name, type, argsArray, comment = '', returnComment = '', required = true) {
```

Comment:
can we have returnComment the same way we have arguments?

Reply:
while this makes the return code nicer, it makes the .type code uglier.